service/controller/v25/controllercontext/status.go

```diff
@@ -35,16 +35,14 @@ type ContextStatusControlPlaneVPC struct {
 }
 
 type ContextStatusTenantCluster struct {
-	AWSAccountID           string
-	EncryptionKey          string
-	HostedZoneNameServers  string
-	KMS                    ContextStatusTenantClusterKMS
-	MasterInstance         ContextStatusTenantClusterMasterInstance
-	TCCP                   ContextStatusTenantClusterTCCP
-	VersionBundleVersion   string
-	VPC                    ContextStatusTenantClusterVPC
-	VPCPeeringConnectionID string
-	WorkerInstance         ContextStatusTenantClusterWorkerInstance
+	AWSAccountID          string
+	EncryptionKey         string
+	HostedZoneNameServers string
+	KMS                   ContextStatusTenantClusterKMS
+	MasterInstance        ContextStatusTenantClusterMasterInstance
+	TCCP                  ContextStatusTenantClusterTCCP
+	VersionBundleVersion  string
+	WorkerInstance        ContextStatusTenantClusterWorkerInstance
 }
 
 type ContextStatusTenantClusterKMS struct {
```

Comment:
What does KMS mean in this context?

There is already the AWS KMS service we use for encryption. But this looks different. 😕

Reply:
There is the encryptionkey resource and the kmskeyarn resource. Both work towards the encryption. One fetches the key, one the ARN. The key is used for cloud config encryption. The ARN is used with the IAM policies to allow the encryption. We could move fetching the information to one resource, but this is out of scope here.

Reply:
But why are the fields all to do with networking? I'd expect to see the KMS key name or the ARN.

My aws-operator knowledge is a bit rusty but that is why I'm confused.

Reply:
Oh wow its just a weird diff! Checked the file and I get it now.